.github/stale.yml

```diff
@@ -0,0 +1,60 @@
+---
+# Configuration for probot-stale - https://github.com/probot/stale
+
+# Number of days of inactivity before an Issue or Pull Request with the stale
+# label is closed. Set to false to disable. If disabled, issues still need to
+# be closed manually, but will remain marked as stale.
+daysUntilClose: 7
+
+# Set to true to ignore issues in a project (defaults to false)
+exemptProjects: false
+
+# Set to true to ignore issues in a milestone (defaults to false)
+exemptMilestones: false
+
+# Set to true to ignore issues with an assignee (defaults to false)
+exemptAssignees: false
+
+# Label to use when marking as stale
+staleLabel: stale
+
+# Comment to post when removing the stale label.
+# unmarkComment: >
+
+closeComment: >
+  This issue/pull request has been automatically closed due to inactivity and
+  no response to the auto close bot request. If you feel this is
+  inappropriate, please respond to this issue requesting it to be re-opened.
+
+# Limit the number of actions (marking or closing issues/PRs) per hour, from
+# 1-30. Default is 30
+limitPerRun: 30
+
+# Issue specific handling:
+issues:
+  daysUntilStale: 60
+  markComment: >
+    This issue has been automatically marked as stale because it has not had
+    recent activity. It will be closed if no further activity occurs. Thank you
```

Comment:
Should we say `closed in 7 days`  to make it clear what the timeline is? Hmm, I wonder if you can use some sort of `{{yaml substitution}}` in there to pick up the value actually configured?

Reply:
I see what you're saying, but I hate hard-coding numbers like that. There doesn't seem to be a way to refer to `$time_until_thing_will_be_closed` in these blocks and since it's actually `30+7` days for a PR and `60+7` days for an issue, if an issue/PR has been left *that* long, it seems unlikely the author will be coming back. But by not specifying the time left, we add a frisson of tension which _might_ entice them back :smile: